docs/index.rst

```diff
@@ -229,6 +229,8 @@ Ready to start reviewing pull requests? We have :doc:`a guide </reviewing_PRs>`
 for that too! Finally, the :doc:`API reference </api/index>` covers individual
 functions, classes, methods and modules.
 
+**Robottelo** is compatible with Python 2.7.
```

Comment:
I'm running robottelo with python 2.7.5. Any one tried with python3.5 ? If so and didn't see any issue with that then please update that too.